python/tvm/relay/frontend/tflite.py

```diff
@@ -1033,6 +1034,35 @@ def convert_split(self, op):
 
         return out
 
+    def convert_slice(self, op):
+        """Convert TFLite SLICE"""
+        try:
+            from tflite.Operator import Operator
+        except ImportError:
+            raise ImportError("The tflite package must be installed")
+
+        assert isinstance(op, Operator)
+        input_tensors = self.get_input_tensors(op)
+        assert len(input_tensors) == 3, "input tensors length should be == 3"
+        input_tensor = input_tensors[0]
+        in_expr = self.get_expr(input_tensor.tensor_idx)
+
+        begin = list(self.get_tensor_value(input_tensors[1]))
+        size = list(self.get_tensor_value(input_tensors[2]))
+        # strided_slice(Relay) needs the slice's end indices, not the size
+        end = size
+        input_tensor_shape = input_tensor.tensor.ShapeAsNumpy()
+        input_tensor_rank = len(input_tensor_shape)
+        for i in range(input_tensor_rank):
+            if size[i] == -1:
+                end[i] = input_tensor_shape[i] - begin[i] + 1
```

Comment:
The `+1` is correct? see tf frontend: https://github.com/apache/incubator-tvm/blob/master/python/tvm/relay/frontend/tensorflow.py#L627 Let us make sure again. If tf is wrong, please fix tf frontend.

Reply:
Thanks for reviewing! I will fix the TF parser as soon as we agree on the correctness of the formula.

Reply:
@FrozenGene, the PR fixing the TF parser is ready: #4518

Reply:
In tf doc, when size=-1. it means all remaining elements should be included. So here should be end[i] = data_shape[i]?

Reply:
@kevinthesun I see what you mean, but **end[i]** is zero-based while **data_shape** is one-based so following your logic, end[i] = data_shape[i] **- 1**

Reply:
end[i] is the last element which not included in the slice. size = -1 requires all remaining elements included. If end[i] = data_shape[i] - 1, the last element is dropped.

Reply:
@kevinthesun ok, I agree with you. I changed it.